lang/cs/Org.Apache.REEF.Tang/Interface/IConfigurationBuilder.cs

```diff
@@ -22,7 +22,7 @@ namespace Org.Apache.REEF.Tang.Interface
 {
     public interface IConfigurationBuilder
     {
-        void AddConfiguration(IConfiguration c); 
+        void AddConfiguration(IConfiguration c);
```

Comment:
I thought git would automagically remove it from PR
I'll look into it

---
In reply to: [86018763](https://github.com/apache/reef/pull/1161#discussion_r86018763) [](ancestors = 86018763)